django/http/request.py

```diff
@@ -461,6 +461,13 @@ def parse_header_name(cls, header):
             return None
         return header.replace("_", "-").title()
 
+    @classmethod
+    def to_header_name(cls, header):
```

Comment:
I get that this is `to_header_name()` as a sort of opposite to `parse_header_name()` but I find the name confusing. I'd say that, for example, `X-Header-Name` was the usual form of the header name and not `HTTP_X_HEADER_NAME`.

We likely can't change `parse_header_name()` now, and it's not the worst - it still sort of makes sense - but we could change `to_header_name()` to something like `to_meta_name()` or `format_meta_name()`? Or even `unparse_header_name()`? 🤔

Reply:
Now that I've been looking at the ASGI equivalent below, I feel that we should perhaps call this `to_wsgi_name()` instead...

Also, for this helper method we're only using it in one place and I think we could push that loop in here and call it `to_wsgi_names()` instead (or provide that in addition).